widgets/views/templateSelectModal.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use humhub\widgets\Button;
+use humhub\widgets\Modal;
+use humhub\widgets\ModalButton;
+use yii\helpers\Html;
+use humhub\modules\wiki\helpers\Url;
+use yii\widgets\ActiveForm;
+
+$templateOptions = [];
+
+foreach ($templates as $template) {
+    $templateOptions[$template->id] = Html::encode($template->title);
+}
+
+?>
+
+<?= Modal::widget([
+    'id' => 'templateSelectModal',
+    'header' => '<strong>New page</strong>',
+    'body' => '<div class="text-center">' .Yii::t('WikiModule.base', '<p>Start with an empty page or template<p>') .
+                Button::primary(Yii::t('WikiModule.base', 'Create blank page'))->id('blankPageBtn')->loader(false) .
+                '<div class="d-flex align-items-center my-3">
+                    <span class="mx-2 text-muted">OR</span>
+                    <hr class="flex-grow-1">
+                </div>' .
+            '</div>',
+    'footer' => 
+        '<select id="templateSelectDropdown" name="templateSelectDropdown" class="form-control" style="margin-bottom: 20px;">' .
+        '<option value="">' . Yii::t('WikiModule.base', 'List of templates') . '</option>' .
```

Comment:
`List of templates` should not be a dropdown option itself, Instead should be just a hint.
If this is not possible, you could just pre-select the first option.

In addition, please a German translation as well, e.g. "Liste der Templates"